### PR TITLE
Update timetest tool to support ip and op params config

### PR DESCRIPTION
**Pull request by pereanub**
### Details:
 - *User can set input and output precision for the timetest tool*

### Tickets:
 - *[CVS-104240](https://jira.devtools.intel.com/browse/CVS-104240)*

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
why do we need to test such scenarios? Users may also want to add a lot of things to the model, but it does not mean we need it in this tool.

**Comment by pereanub** [comment]
> why do we need to test such scenarios? Users may also want to add a lot of things to the model, but it does not mean we need it in this tool.

Because we found a specific case with IRv10 where OpenVino is changing precision from FP16 to FP32 even if that network has input and output set to FP16 and that creates some issues when running it on VPU. The problem is described in [CVS-104240](https://jira.devtools.intel.com/browse/CVS-104240).

If you have other suggestions on how it can be fixed, please let us know.

**Comment by ilya-lavrenov** [comment]
> if that network has input and output set to FP16

how is it set? Could you please provide more details in which flow OpenVINO changes precisions? 
Is the model originally in fp16? converted via MO? etc

**Comment by pereanub** [comment]
> > if that network has input and output set to FP16
> 
> how is it set? Could you please provide more details in which flow OpenVINO changes precisions? Is the model originally in fp16? converted via MO? etc

Originally the model is FP16, and OpenVino change it to FP32 when read_model is called. Please check the ticket for more details.

**Review by vurusovs — CHANGES_REQUESTED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Review by vurusovs — APPROVED** [review]

File: tests/lib/src/common_utils.cpp [hunk 1]
```diff
@@ -65,3 +65,34 @@ void fillBlobs(InferenceEngine::InferRequest inferRequest,
...
+        {"U64", ov::element::u64},
+        {"I64", ov::element::i64},
+        {"U32", ov::element::u32},
+        {"I32", ov::element::i32},
+        {"U16", ov::element::u16},
+        {"I16", ov::element::i16},
+        {"U8", ov::element::u8},
+        {"I8", ov::element::i8},
+        {"BOOL", ov::element::boolean},
+    };
```
[the remark this answers is not in the document]
Comment (ilya-lavrenov) [reply tests/lib/src/common_utils.cpp:86]: created ticket CVS-107170

**Pull request merged**